ng2-components/ng2-alfresco-documentlist/README.md

```diff
@@ -169,24 +168,25 @@ platformBrowserDynamic().bootstrapModule(AppModule);
 
 | Name | Type | Default | Description |
 | --- | --- | --- | --- |
-| `navigate` | boolean | true | Toggles navigation to folder content or file preview |
-| `navigationMode` | string (click\|dblclick) | dblclick | User interaction for folder navigation or file preview |
-| `thumbnails` | boolean | false | Show document thumbnails rather than icons |
-| `fallbackThubnail` | string |  | Fallback image for row ehre thubnail is missing|
-| `multiselect` | boolean | false | Toggles multiselect mode |
-| `contentActions` | boolean | false | Toggles content actions for each row |
-| `contextMenuActions` | boolean | false | Toggles context menus for each row |
-| `rowFilter` | `RowFilter` | | Custom row filter, [see more](#custom-row-filter).
-| `imageResolver` | `ImageResolver` | | Custom image resolver, [see more](#custom-image-resolver).
+| rootPath | string | -root- | Root node path, i.e. `-root-`, `-shared-`, `-my-`, etc. |
```

Comment:
Can you leave the style like that  `navigate` in all the table?.
This style is equal across all the components. We are trying to have a common style in the readme if we want to modify this style we have to modify that in all the readme.